appserver/packager/external/tomlj-repackaged/pom.xml

```diff
@@ -0,0 +1,126 @@
+  ~  enclosed by brackets [] replaced by your own identifying information:
+  ~  "Portions Copyright [year] [name of copyright owner]"
+  ~
+  ~  Contributor(s):
+  ~  If you wish your version of this file to be governed by only the CDDL or
+  ~  only the GPL Version 2, indicate your decision by adding "[Contributor]
+  ~  elects to include this software in this distribution under the [CDDL or GPL
+  ~  Version 2] license."  If you don't indicate a single choice of license, a
+  ~  recipient has the option to distribute your version of this file under
+  ~  either the CDDL, the GPL Version 2 or to extend the choice of license to
+  ~  its licensees as provided above.  However, if you add GPL Version 2 code
+  ~  and therefore, elected the GPL Version 2 license, then the option applies
+  ~  only if the new code is made subject to such option by the copyright
+  ~  holder.
+  ~
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>fish.payara.server.internal.packager</groupId>
+        <artifactId>external</artifactId>
+        <version>6.2024.11-SNAPSHOT</version>
+    </parent>
+    <artifactId>tomlj-repackaged</artifactId>
+    <name>tomlj repackaged as a module</name>
+    <description>tomlj repackaged as a module</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <tomlj.version>1.1.1</tomlj.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-jar</id>
+                        <configuration>
+                            <skipIfEmpty>true</skipIfEmpty>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <instructions>
+                        <Embed-Dependency>
+                            *;scope=compile;inline=true
+                        </Embed-Dependency>
+                        <Export-Package>
+                            !org.antlr.*,!org.checkerframework.checker.nullness.qual,!com.ibm.icu.*,!org.abego.treelayout.*,!org.stringtemplate.*,*;version=${tomlj.version}
+                        </Export-Package>
+                        <Import-Package>
+                            !org.antlr.*,!org.checkerframework.checker.nullness.qual,!com.ibm.icu.*,!org.abego.treelayout.*,!org.stringtemplate.*,*
```

Comment:
ICU4J is licensed under unicode. This is not currently under our approved license list.
This will need reverifying if it's a compatible license.

Reply:
No longer used